### PR TITLE
Travis: Only run lintr and covr if R release and push

### PR DESCRIPTION
**Pull request by arnfinn**
To avoid lintr echo

### CODE REVIEW

File: R/shiny_functions.R [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #' @export
 #'
 
-qi_table <- function(table_data, enhet) {
+qi_table <- function(table_data, enhet){
```
Comment (arnfinn) [inline R/shiny_functions.R]: Will fix this before merge. Only for testing `lintr`

Comment (arnfinn) [inline R/shiny_functions.R]: Will fix this before merge. Only for testing `lintr`

**Comment by areedv** [comment]
[R/shiny_functions.R:9:39:](https://github.com/SKDE-Felles/qmongr/blob/c3e2293334c36aeeaa5d39a2583773b4111204b4/R/shiny_functions.R#L9) *style:* **There should be a space between right parenthesis and an opening curly brace.**
```r
​qi_table <- function(table_data, enhet){
                                      ^~
```

**Comment by arnfinn** [comment]
Not the best time for testing this because of https://www.githubstatus.com/incidents/xp2qc958g4wt

**Comment by areedv** [comment]
[R/shiny_functions.R:9:39:](https://github.com/SKDE-Felles/qmongr/blob/9b8fb991fa9fab7f3055bd6fe81b6995aae241db/R/shiny_functions.R#L9) *style:* **There should be a space between right parenthesis and an opening curly brace.**
```r
​qi_table <- function(table_data, enhet){
                                      ^~
```
[R/shiny_functions.R:10:14:](https://github.com/SKDE-Felles/qmongr/blob/9b8fb991fa9fab7f3055bd6fe81b6995aae241db/R/shiny_functions.R#L10) *style:* **Do not place spaces around code in parentheses or square brackets.**
```r
​  tags$table( 
             ^
```
[R/shiny_functions.R:10:14:](https://github.com/SKDE-Felles/qmongr/blob/9b8fb991fa9fab7f3055bd6fe81b6995aae241db/R/shiny_functions.R#L10) *style:* **Trailing whitespace is superfluous.**
```r
​  tags$table( 
             ^
```
[R/shiny_functions.R:11:16:](https://github.com/SKDE-Felles/qmongr/blob/9b8fb991fa9fab7f3055bd6fe81b6995aae241db/R/shiny_functions.R#L11) *style:* **Do not place spaces around code in parentheses or square brackets.**
```r
​    tags$thead( 
               ^
```
[R/shiny_functions.R:11:16:](https://github.com/SKDE-Felles/qmongr/blob/9b8fb991fa9fab7f3055bd6fe81b6995aae241db/R/shiny_functions.R#L11) *style:* **Trailing whitespace is superfluous.**
```r
​    tags$thead( 
               ^
```
[R/shiny_functions.R:12:15:](https://github.com/SKDE-Felles/qmongr/blob/9b8fb991fa9fab7f3055bd6fe81b6995aae241db/R/shiny_functions.R#L12) *style:* **Do not place spaces around code in parentheses or square brackets.**
```r
​      tags$tr( 
              ^
```
[R/shiny_functions.R:12:15:](https://github.com/SKDE-Felles/qmongr/blob/9b8fb991fa9fab7f3055bd6fe81b6995aae241db/R/shiny_functions.R#L12) *style:* **Trailing whitespace is superfluous.**
```r
​      tags$tr( 
              ^
```

File: .travis.yml [hunk 2]
```diff
@@ -26,10 +26,13 @@ before_install:
 
 after_success:
   - R CMD INSTALL $PKG_TARBALL
-  - Rscript -e 'lintr::lint_package()'
+  - |
```
Comment (areedv) [inline .travis.yml]: Hva betyr egentlig denne?

Comment (arnfinn) [reply .travis.yml]: https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

"Values can span multiple lines using | or >."

**Pull request merged**